### PR TITLE
ODATA-1136 - resolved - Allow hierarchy filter functions for ancestors and descendants ...

### PR DESCRIPTION
**Pull request by GeraldKrause**
Allow hierarchy filter functions for ancestors and descendants to return true for starting node

### CODE REVIEW

**Review by HeikoTheissen — APPROVED** [review]

File: vocabularies/Org.OData.Aggregation.V1.xml [hunk 1]
```diff
@@ -214,21 +214,35 @@
 
       <Function Name="isdescendant" IsBound="true">
         <Annotation Term="Core.Description"
-          String="Returns true, if and only if the value of the node property of the specified hierarchy is a descendant of the given parent node with a distance of less than or equal to the optionally specified maximum distance" />
+          String="Returns true, if and only if the value of the node property of the specified hierarchy is a descendant of the given parent node with a distance of less than or equal to the optionally specified maximum distance or, if selected, optionally the parent node itself" />
```
Comment (ralfhandl) [inline vocabularies/Org.OData.Aggregation.V1.xml:217]: I find the "if selected" unprecise/confusing.

**Review by ralfhandl — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]
Have already merged this into #125.

**Comment by ralfhandl** [comment]
> Have already merged this into #125.

Do we want to merge this PR now, or just close it?

**Comment by HeikoTheissen** [comment]
> @HeikoTheissen, @GeraldKrause: do we want to merge this PR now, or just close it?

Better merge it, because I have changed some descriptions in #125. The diff of #125 shows then the old vs. the new descriptions.

**Pull request merged**